p5/core/font.py

```diff
@@ -99,7 +88,10 @@ def text(*args, wrap_at=None):
 
     if len(text_string) == 0:
         return
-    return p5.renderer.text(text_string, position, wrap_at)
+    if builtins.current_renderer == "vispy":
+        return p5.renderer.text(text_string, position, wrap_at)
+    elif builtins.current_renderer == "skia":
+        return p5.renderer.text(text_string, *position, wrap_at)
```

Comment:
Is there a reason to make the text API different between renderers? I think we should make them consistent if possible.

Reply:
I think we can adjust the skia renderer to remove the if-else loops.